mappings/net/minecraft/server/network/packet/ClientSettingsC2SPacket.mapping

```diff
@@ -1,16 +1,19 @@
 CLASS net/minecraft/class_2803 net/minecraft/server/network/packet/ClientSettingsC2SPacket
 	FIELD field_12777 language Ljava/lang/String;
 	FIELD field_12778 playerModelBitMask I
+	FIELD field_12779 chatColors Z
 	FIELD field_12780 viewDistance I
 	FIELD field_12781 chatVisibility Lnet/minecraft/class_1659;
 	FIELD field_12782 mainArm Lnet/minecraft/class_1306;
 	METHOD <init> (Ljava/lang/String;ILnet/minecraft/class_1659;ZILnet/minecraft/class_1306;)V
 		ARG 1 language
 		ARG 2 viewDistance
 		ARG 3 chatVisibility
+		ARG 4 chatColors
 		ARG 5 modelBitMask
 		ARG 6 mainArm
 	METHOD method_12131 getLanguage ()Ljava/lang/String;
 	METHOD method_12132 getMainArm ()Lnet/minecraft/class_1306;
 	METHOD method_12134 getChatVisibility ()Lnet/minecraft/class_1659;
+	METHOD method_12135 getChatColors ()Z
```

Comment:
No `getXxx` for boolean getters

Reply:
`areChatColorsEnabled` may be a good alternative

Reply:
`hasChatColors` or `allowsChatColors`
```suggestion
	METHOD method_12135 hasChatColors ()Z
```